### PR TITLE
feat: refactor develcoSpecificAirQuality into manuSpecificDevelcoAirQuality

### PR DESCRIPTION
**Pull request by sjorge**
Cleanup some of the develco stuff:
- [x] move clusters from zh to zhc
- [x] move attributes from zh to zhc
- [x] convert the sw/hw reading into modernExtend
- [x] move some of the specific attrbute based converters to modernExtend

Merge before https://github.com/Koenkk/zigbee-herdsman/pull/1097

### CODE REVIEW

File: src/lib/develco.ts [hunk 1]
```diff
@@ -7,6 +7,40 @@ import {logger} from './logger';
 const NS = 'zhc:develco';
 
 export const develcoModernExtend = {
+    addCustomClusterManuSpecificDevelcoGenBasic: () => deviceAddCustomCluster(
+        'genBasic',
+        {
+            ID: 0x0000,
+            attributes: {
+                zclVersion: {ID: 0, type: Zcl.DataType.UINT8},
```
Comment (sjorge) [inline src/lib/develco.ts:15]: @Koenkk The merging of these is not 100% clear to do.

Can I just not repeat all the ZCL genBasic attributes here and only add the 3 develco ones (L30-L32) here and have it properly merged?

Comment (Koenkk) [reply src/lib/develco.ts:16]: It will get merged by default, so no need to re-specify everything.

**Comment by sjorge** [comment]
I might need to start over, i can't seem to resolve the merge conflict :(

**Comment by sjorge** [comment]
Going to test this a bit more and then mark ready, there is some more cleanup that can be done but I'm trying to work out how to best do it.

I think some of these can be a more generic, like a lot of them are ignoring certain bogus values. Which can probably be added in the generic modernExtend as it could be useful for other devices. I'll need to think about this some more.

File: src/lib/develco.ts [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        numeric({
+            name: 'voc',
+            cluster: 'manuSpecificDevelcoAirQuality',
+            attribute: 'measuredValue',
+            reporting: {min: '1_MINUTE', max: '1_HOUR', change: 10},
+            description: 'Measured VOC value',
+            // from Sensirion_Gas_Sensors_SGP3x_TVOC_Concept.pdf
+            // "The mean molar mass of this mixture is 110 g/mol and hence,
+            // 1 ppb TVOC corresponds to 4.5 μg/m3."
+            scale: 4.5,
```
Comment (sjorge) [inline src/lib/develco.ts:77]: @Koenkk klopt dit?

https://github.com/Koenkk/zigbee-herdsman-converters/pull/7706/files#diff-97dc2d48d7db27dd10bb142fd276583a4dfdff437808452d813cfb4bb65b2b59L161 het zou dit moeten doen dus `* 4.5` maar het lijkt eerder een `/ 4.5` te doen?

Comment (Koenkk) [reply src/lib/develco.ts]: Scale doet een `/`: https://github.com/Koenkk/zigbee-herdsman-converters/blob/a60047c170b562be05888513a9c493143ad91d7f/src/lib/modernExtend.ts#L1869

**Comment by sjorge** [comment]
@Koenkk Dit kan al gemerged worden, de commits waar ik het in discord over heb waar ik nog tegen problemen loop zitten hier niet bij. En dit moet daar niet op wachten.

File: src/lib/develco.ts (hunk 2 above)
Comment (sjorge) [reply src/lib/develco.ts]: Found a ScaleFunction did the trick nicely.

**Comment by Koenkk** [comment]
Bedankt!

**Pull request merged**